stockApp/stockApp/Presentation/ViewModel/AssetViewModel.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+// MARK: - Class
+
+class AssetViewModel: AssetViewModelProtocol {
+
+    internal let useCase: GetAssetUseCaseProtocol
```

Comment:
```suggestion
    let useCase: GetAssetUseCaseProtocol
```

if yo don't have frameworks, always it's going to be internal..

Reply:
done